esterni/glossario/glossario.tex

```diff
@@ -29,212 +32,209 @@
+		\item[Stima:] previsione e valutazione del valore economico e del tempo da impiegare nell'ambito di un progetto.
+		\item[Subfiles:] pacchetto di LaTeX che fornisce un modo per compilare le sezioni di un documento usando lo stesso preambolo del documento principale.
+		\item[Swagger:] framework software open source supportato da un vasto ecosistema di strumenti che aiuta gli sviluppatori a progettare, costruire, documentare e utilizzare servizi Web RESTful.
 
-  \end{description}
-  \newpage
-  \section{W}
-  \begin{description}
-    \item[Way of working:] modo di lavorare del team caratterizzato dall'utilizzo di best practices. Esso viene raffinato nel tempo attraverso la manutenzione migliorativa.
-    \item[Web application:] applicazione distribuita accessibile in una rete Internet per mezzo di un'architettura client/server.
-  \end{description}
-  \newpage
-  \section{Y}
-  \begin{description}
-    \item[YAML:] standard di serializzazione dei dati a misura d'uomo per tutti i linguaggi di programmazione.
 	\end{description}
+	\newpage
+	\section{T}
+	\begin{description}
+		\item[Tag:] aspetto di Git che permette di identificare una versione di una release specifica del codice prodotto.
+		\item[Technology Baseline:] definisce le tecnologie, i framework e le librerie selezionate per lo sviluppo del prodotto.
+		\item[Telegram:] servizio di messaggistica istantanea e broadcasting basato sul cloud.
+		\item[Test end-to-end:] metodologia utilizzata per verificare se il flusso di un'applicazione sta funzionando come progettato dall'inizio alla fine.
+		\item[Ticket:] termine tecnico utilizzato per riferirsi all'assegnazione della responsabilità di una determinata issue su GitHub ad un membro del team di progetto.
+		\item[Tracciatura:] rilevamento della presenza di un'utente autenticato all'interno di un luogo. Può essere anonima quando l'utente non è autenticato nell'organizzazione di riferimento, oppure autenticata quando l'utente è autenticato nell'organizzazione di riferimento.
+	\end{description}
+	\newpage
+	\section{U}
+	\begin{description}
+		\item[UML:] acronimo di Unified Modeling Language, rappresenta un linguaggio visuale che si basa su un insieme di regole, vincoli e teorie utilizzate per la modellazione di una classe di problemi. Serve a supportare la descrizione dei sistemi software. La versione corrente di UML è la 2.0.
+		\item[Utente:] soggetto che ha interesse ad utilizzare il software prodotto da GruppOne.
+		\item[Utente autenticato:] utente non autenticato che ha effettuato l'autenticazione alla piattaforma dall'applicazione mobile oppure alla web application.
+		\item[Utente collegato:] utente autenticato che è collegato ad almeno un'organizzazione;
+		\item[Utente collegato incognito:] utente collegato ad un'organizzazione in modalità incognito, in quanto la sua identità non è conosciuta.
+		\item[Utente collegato noto:] utente collegato ad un'organizzazione in modalità noto, in quanto la sua identità è conosciuta.
```

Comment:
modalità noto o modalità autenticata?

Reply:
L'utente è autenticato sia in modalità incognito che in modalità noto. Quindi rimane modalità noto